GDTEST_ANALYZER_ERROR
>> ERROR at line 3: Cannot have a key of type "int" in a dictionary of type "Dictionary[Object, Object]".
